### ISSUE TITLE
LMAP support incomplete

### ISSUE THREAD

**Issue by hmh** [schoenw/lmapd#10]
lmapd does not support (or implement) all of the functionality present in the RFCs.  The SIMET team from NIC.br will need some of the missing functionality, so we plan to implement them at least partially in the simetnicbr/simet-lmapd fork.

- [x] ietf-lmap-report::result::table::function
- [x] ietf-lmap-report::result::table::column
- [ ] ietf-lmap-report::result::conflict (low priority for us)
- [ ] ietf-lmap-control::schedule::action::parameters and ietf-lmap-report::result::parameters (only if we require it for scheduler extensions)

(we will edit this issue as we identify more functionality that we need to implement).

**Comment by hmh** [comment]
We have a PR adding result.table.function* and result.table.column mostly ready and will submit it soon, but it depends on PR #11 and also on the code that addresses issue #7.

**Comment by hmh** [comment]
Patches have been made public in our work-in-progress "devel" branch in simetnicbr/simet-lmapd. We have also split them per-functionality where possible, as "for-upstream/*" branches.

**Comment by hmh** [comment]
Now this depends on PR #18 instead of #11.
